### PR TITLE
tweaked bins + added bin/phpunit-package

### PR DESCRIPTION
**Pull request by ro0NL**

### CODE REVIEW

File: bin/cs [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+    DIR="src/$(echo $1 | sed -r 's/(^|-)(\w)/\U\2/g')"
+    shift;
+fi
+
 BIN="vendor/bin/php-cs-fixer"
-CMD="${BIN} fix --dry-run --verbose --diff ${@:-src/}"
+CMD="${BIN} fix --dry-run --verbose --diff ${@} ${DIR}"
 
 if [[ ! -f $BIN ]] ; then
     composer update --no-progress --prefer-dist --no-interaction --no-suggest
```
Comment (GawainLynch) [inline bin/cs]: `[[ $? -ne 0 ]] && exit 1`

File: bin/create-project [hunk 2]
```diff
@@ -46,3 +46,14 @@ if [[ ! -z $BUNDLES ]] ; then
 fi
 
 popd &> /dev/null
+
+echo -en "\e[34mLink dependencies? [yN]\e[0m "
+read answer
+if [[ $answer =~ ^y|Y|yes|YES$ ]] ; then
+    composer link -h &> /dev/null
+    if [[ $? != 0 ]] ; then
+        composer require --dev --no-progress --prefer-dist --no-interaction --no-suggest ro0nl/link
```
Comment (GawainLynch) [inline bin/create-project]: `[[ $? -ne 0 ]] && exit 1`

File: bin/phpunit-package [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+fi
+
+if [[ ! -f "${PACKAGE_DIR}/${BIN}" ]] ; then
+    composer update --no-progress --prefer-dist --no-interaction --no-suggest --working-dir="${PACKAGE_DIR}"
+fi
+
+if [[ ! $TRAVIS ]] || ( [[ $DEPS = high && $TRAVIS_BRANCH = master ]] ) ; then
+    composer link -h &> /dev/null
+    if [[ $? != 0 ]] ; then
+        composer require --dev --no-progress --prefer-dist --no-interaction --no-suggest ro0nl/link
```
Comment (GawainLynch) [inline bin/phpunit-package]: `[[ $? -ne 0 ]] && exit 1`

File: bin/sa [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
+    DIR="src/$(echo $1 | sed -r 's/(^|-)(\w)/\U\2/g')"
+    shift;
+fi
+
 BIN="vendor/bin/phpstan"
-CMD="${BIN} analyse --configuration phpstan.neon --level max ${@:-src/}"
+CMD="${BIN} analyse --configuration phpstan.neon --level max ${@} ${DIR}"
 
 if [[ ! -f $BIN ]] ; then
     composer update --no-progress --prefer-dist --no-interaction --no-suggest
```
Comment (GawainLynch) [inline bin/sa]: `[[ $? -ne 0 ]] && exit 1`

**Review by GwendolenLynch — COMMENTED** [review]
Other than catching those return failure, LGTM

**Comment by ro0NL** [comment]
Should be good! exit codes added below your comments :(

**Review by GwendolenLynch — APPROVED** [review]

**Pull request merged**